config.ru

```diff
@@ -1,10 +1,8 @@
-$:.unshift("lib")
+require 'i18n_yaml_editor'
 
-require "i18n_yaml_editor/app"
-require "i18n_yaml_editor/web"
+iye_app = I18nYamlEditor::App.new('example')
+iye_app.load_translations
```

Comment:
We're doing `load_translations/create_missing_keys` multiple places. Put in a single method?